Portfolio/index.html

```diff
@@ -13,18 +14,22 @@
     <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
     <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
     <nav>
+    <h2>Jahnavi</h2>
         <ul>
             <li>Home</li>
             <li>About</li>
             <li>Skills</li>
             <li>Contact</li>
         </ul>
     </nav>

```

Comment:
Consider adding text or labels before the social media icons to provide context to the users.